index.js

```diff
@@ -19,8 +19,23 @@ module.exports = function (session) {
 
     this.conString = options.conString || process.env.DATABASE_URL;
     this.ttl = options.ttl;
-    this.pg = options.pg || require('pg');
-    this.ownsPg = !options.pg;
+
+    if (options.pgPool !== undefined) {
+      this.pgPool = options.pgPool;
+      this.ownsPg = false;
+    } else {
+      if (options.pg !== undefined) {
+        this.pg = options.pg;
+        this.ownsPg = false;
+      } else {
+        this.pg = require('pg');
+        this.ownsPg = true;
+      }
+      if (typeof this.pg.Pool === 'function') {
+        throw new Error('The pgPool option must be supplied to the ' +
+            'connect-pg-simple constructor when using pg version 6 or above');
+      }
+    }
```

Comment:
First of all:

Rename `pgPool` to just `pool`, the `pg`-part is fairly redundant. Do this both in `this.` and `options.`

Second:

I think it makes sense that this new version of drops supports for older `pg`-versions – people who need to use an older `pg` module can use an older version of this module and it keeps the code clean going forward.

That said – we can still support establishing a connection when no `pool` has been sent it. We can still support the value of `this.conString`, bur rather than assigning it as a property we can use it to create a pool ourselves. So when no `pool` has been supplied we can use the `options.conString` and default to `process.env.DATABASE_URL` if no `options.conString` has been supplied and with that data create a new `pg.Pool` using our own `pg` module.

That way the *Simple example:* in the README will keep working the same, but be using the new pg module.